templates/fine-tune-llm_v2/training_configs/full_param/mistral-7b.yaml

```diff
@@ -11,10 +11,8 @@ num_checkpoints_to_keep: 1
 dataset_size_scaling_factor: 10000
 output_dir: /mnt/local_storage
 deepspeed:
-  config_path: deepspeed_configs/zero_3_llama_2_13b.json
+  config_path: deepspeed_configs/zero_3_offload_optim+param.json
 dataset_size_scaling_factor: 10000 # internal flag. No need to change
 flash_attention_2: true
```

Comment:
why remove the `trainer_resouces` ?

Reply:
This ensures the trainer worker picks the large g5 nodes so that it doesn't OOM during checkpointing.

Reply:
I believe this was there because the config was previously for Llama 2 13 with 4k context length. I renamed it and reused it for Mistral now (llama 3 8B works without this line as well)

Reply:
But the one for llama-3-8B still has this memory requirement. It may work because you are giving it the gpu resources you need. If you don’t specify mem requirements it may get smallest g5 node.

Reply:
I don't see the line you are referring to. Looking at the Llama 3 8B configs I don't see this.

But you are still right, I did another round of testing and have updated the compute configs.
I trusted the worker and trainer resources there but started OOMing on super small instances.
I have updated some configs based on using 16 (or 32) small nodes that did not OOM.